### PR TITLE
feat: delegate native logics to Finschia/cosmos-sdk

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

This patch would remove native logic from this repository and bump cosmos-sdk to v0.50.2.

closes: #1204

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by 0Tech** [comment]
We will ignore some workflow errors for now.

**Review by jaeseung-bae — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,360 +29,46 @@ Types of changes (Stanzas):
...
 -->
 
 # Changelog
 
-## [Unreleased]
-
-## [v0.50.1](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.1) - 2023-11-07
-
-> v0.50.0 has been retracted due to a mistake in tagging the release. Please use v0.50.1 instead.
+## [Unreleased](https://github.com/Finschia/finschia-sdk/compare/v0.48.0...HEAD)
```
Comment (jaeseung-bae) [inline CHANGELOG.md:40]: Not from v0.48.1 which is latest version of finschia-sdk?

Comment (0Tech) [reply CHANGELOG.md:40]: This feature branch is based on main, so it's not from v0.48.1.

**Comment by 0Tech** [comment]
I've updated workflows recently. We will also update workflows on subsequent PRs, but these are more or less essential at this point, because there could be side effects. I request the reviewers to take special care for the workflows.

**Review by tkxkd0159 — APPROVED** [review]

**Pull request merged**